src/main.ts

```diff
@@ -555,18 +589,12 @@ async function getAllActionsUsingSearch(
 }
 
 async function getForkParent(
-  client: Octokit,
-  owner: string,
-  repo: string
+  repoDetails: any,
```

Comment:
I dislike the use of `any` here, but do no see a way to improve this method 😞.

Reply:
I also could not find a type to use, for now let it what it is ?

Reply:
Exactly. Leave it in as is for now.